src/Microsoft.MobileBlazorBindings/MobileBlazorBindingsHostExtensions.cs

```diff
@@ -33,10 +33,7 @@ public static class MobileBlazorBindingsHostExtensions
             }
 
             var services = host.Services;
-
-#pragma warning disable CA2000 // Dispose objects before losing scope
-            var renderer = new MobileBlazorBindingsRenderer(services, services.GetRequiredService<ILoggerFactory>());
-#pragma warning restore CA2000 // Dispose objects before losing scope
+            var renderer = services.GetRequiredService<MobileBlazorBindingsRenderer>();
```

Comment:
I like this, can you make it the same at line 70? It's basically the same method called slightly differently but one happens at startup and the other happens on navigation.

Reply:
Actually, do you think we need that method at all? It's not used anymore (since I use renderer directly now).

Reply:
This method is presumably still useful because it's the one that's "actually correct" - that is, it's properly async instead of fire-and-forget. So, let's keep both methods and I'll apply this change to both of them.